daemon/port.go

```diff
@@ -14,8 +14,8 @@ import (
 // }
 
 func Find() (string, error) {
-	maxPort := 9000
-	minPort := 8000
+	maxPort := 90000
+	minPort := 80000
```

Comment:
The port in the code before was inrange between 8000 - 9000. but here i think it was changed.. 
I do think that this high number should be avoided because theoreticaly a system can have 0 ~ 64,000 ports , and that is the standard,  

https://stackoverflow.com/questions/2332741/what-is-the-theoretical-maximum-number-of-open-tcp-connections-that-a-modern-lin

if possible can it be moved to a safe number ?